recipes/catkin/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: catkin
+  version: 0.7.17
+
+source:
+  url: https://github.com/ros-gbp/catkin-release/archive/release/melodic/catkin/0.7.17-0.tar.gz
+  fn: catkin-0.7.17.tar.gz
+  sha256: 316178707bc3f0edcb86a77ec2ec174b98b4d2ccf41306162cefe6588b3a9d41
+  patches:
+    - 001-remove-rt.patch
+    - 002-no-hardcoded-python.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+  host:
+    - python
+    - catkin_pkg
+    - empy
+    - cmake
```

Comment:
``cmake`` should be just on build section

Reply:
this is actually a meta build tool for cmake which needs cmake at run time :)

Reply:
In that case shouldn't it be in `run` instead of `host`?

Reply:
@wolfv Does this point stand?

Reply:
I am moving it to run-depends since the ubuntu debian package also depends on CMake at runtime: https://packages.ubuntu.com/xenial/catkin